packages/mds-daily/db-helpers.ts

```diff
@@ -1,159 +0,0 @@
-import logger from '@mds-core/mds-logger'
-import db from '@mds-core/mds-db'
```

Comment:
Can any/all of the db functions be removed?

Reply:
I don't understand what you mean by "removed" here. Do you mean, can some of these helpers be taken out and used elsewhere?

Reply:
Just wondering if all the mds-db functions that are _only_ used by mds-daily have been deleted.

Reply:
Good call. I just checked and there were a few left that I've now removed.